pubspec.yaml

```diff
@@ -4,17 +4,17 @@ description: A simple Stripe API wrapper, that is meant to be used on the server
 homepage: https://github.com/enyo/stripe-dart
 documentation: https://github.com/enyo/stripe-dart/blob/main/README.md
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ^3.0.0
 dependencies:
-  crypto: ^3.0.0
-  http: ^0.13.1
-  json_annotation: ^4.4.0
-  meta: ^1.1.7
-  logging: ^1.0.1
-  dio: ^5.1.2
+  crypto: ^3.0.3
+  http: ^0.13.6
+  json_annotation: ^4.9.0
+  meta: ^1.15.0
+  logging: ^1.2.0
+  dio: ^5.4.3
 dev_dependencies:
-  json_serializable: ^6.1.4
-  build_runner: ^2.1.7
-  test: ^1.3.0
-  mocktail: ^0.3.0
-  lints: ^1.0.1
+  json_serializable: ^6.8.0
+  build_runner: ^2.4.9
+  test: ^1.25.5
+  mocktail: ^1.0.3
+  lints: ^3.0.0
```

Comment:
These changes don't have to happen now.
The fork is used on several projects, let's play it safe with updates and revert.